b2b-sample/guardio-insurance-sample-application/pages/api/dashboard/me.js

```diff
@@ -16,29 +16,37 @@
  * under the License.
  */
 
-import config from "../../../config.json";
 import getDataHeader from "../../../util/util/apiUtil/getDataHeader";
 import { dataNotRecievedError, notPostError } from "../../../util/util/apiUtil/localResErrors";
+import { getOrgUrl } from "../../../util/util/orgUtil/orgUtil";
 
+/**
+ * backend API call to get logged user details
```

Comment:
`Backend API call to get logged user details.` formatting issues. Check other places too.

Reply:
Fixed with https://github.com/wso2/samples-is/pull/416/commits/8b059f39358c5652ea99e473f4cf34e23ad95e0b